README.md

```diff
@@ -38,11 +38,53 @@ main = runTwilio' (getEnv "ACCOUNT_SID")
   liftIO $ print calls
 
   -- Send a Message.
-  let body = PostMessage "+14158059869" "+14158059869" "Oh, hai"
+  let receivingPhone = "+14158059869"
+  let sendingPhone = "+14158059869"
+  let body = PostMessage receivingPhone sendingPhone "Oh, hai"
   message <- post body
   liftIO $ print message
 ```
 
+Example 2 - using SID and secret embedded in the code
+-------
+
+You can create a REST API client and send a message as follows:
+
+```hs
+{-#LANGUAGE OverloadedStrings #-}
+
+module Main where
+import Twilio.Types.SID
+import Control.Monad.IO.Class (liftIO)
+import Twilio
+import Twilio.Messages
+import qualified Data.Text as T
```

Comment:
Since only `Text` is being used, what do you think about going with a simpler import, e.g.

```hs
import Data.Text (Text)
```

Do you mind alphabetizing the imports as well?